test/ZipStreamTest.php

```diff
@@ -317,7 +317,7 @@ public function testAddFileFromStream(): void
         // it works fine with file stream
         $streamExample = fopen(__FILE__, 'rb');
         $zip->addFileFromStream('sample.txt', $streamExample);
-//        fclose($streamExample);
+        //fclose($streamExample);
```

Comment:
Not relevant to this PR and also ok by the formatters standard.

Reply:
https://github.com/maennchen/ZipStream-PHP/actions/runs/4746949599/jobs/8431234656 tests was failing

Reply:
sorry yes, i was looking in main. i'll make sure to fix the tests.